.github/workflows/update-copyright.yml

```diff
@@ -32,9 +32,18 @@ jobs:
           commitTitle: 'chore: update copyright year in user-facing files (2)'
           token: ${{ secrets.GITHUB_TOKEN }}
           path: |
-            macosx/nl.lproj/InfoPlist.strings
             macosx/Info.plist
             macosx/Info.plist.in
+            macosx/da.lproj/InfoPlist.strings
```

Comment:
As per https://github.com/FantasticFiasco/action-update-license-year
you can use wildcard pattern here.
List of supported languages can be updated in the future.

Reply:
Thanks for the feedback: I had missed the wildcard option. I'll test and update accordingly.